code-reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews at FUN
+
+Code reviews are collaborative steps in software development.
+The purpose of code reviews is twofold. They allow both to:
+* check the quality of the code and
+* to communicate actively on the evolution of the software.
+
+Code reviews concern all members who are involved in software development, regardless of their competence level.

```

Comment:
```suggestion
Code reviews concern all members who are involved in the development team, regardless of their competence level.

```

maybe we can find better, but it's just the term 'software' that doesn't seem relevant in our case.